dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/BaseServiceImpl.java

```diff
@@ -212,4 +212,9 @@ public Map<String, Object> checkAndParseDateParameters(String startDateStr, Stri
         putMsg(result, Status.SUCCESS);
         return result;
     }
+
+    @Override
+    public boolean checkDescriptionLength(String description) {
+        return description!=null && description.codePointCount(0, description.length()) > 255;
```

Comment:
why not just use description.length() > 255?

Reply:
MySQL：In contrast to CHAR, VARCHAR values are stored as a 1-byte or 2-byte length prefix plus data. The length prefix indicates the number of bytes in the value. A column uses one length byte if values require no more than 255 bytes, two length bytes if values may require more than 255 bytes.
Because more than 256 will apply for additional space, lock the table, and then add new fields, it will definitely affect the database performance in the case of high concurrency.